src/config/setupVars.c

```diff
@@ -553,7 +553,7 @@ void importsetupVarsConf(void)
 	get_conf_string_from_setupVars("DHCP_LEASETIME", &config.dhcp.leaseTime);
 
 	// If the DHCP lease time is set to "24", it is interpreted as "24h".
-	// This is some relic from the past that may still be present in some
+	// This is some relict from the past that may still be present in some
```

Comment:
**Note:**
I think `relic` was correct.

Reply:
Reading about it, you are right but this isn't wrong either so I won't undo it.

> Etymological note: both relic and relict are English forms of the past participle, relictus, of the Latin verb relinquere, 'to leave [behind]'.

> Relict is currently used in various academic disciplines as both noun and adjective to mean 'left over', 'survivor' or 'trace of an older state or condition'